.scalafmt.conf

```
@@ -1,16 +1,15 @@
+runner.dialect = scala213
```

It would be worthy to add `version = "3.8.3"` so it's easier to check which version of scalamft we require.
```suggestion
version = "3.8.3"
runner.dialect = scala213
```

---

Done, here and in the other `.scalafmt.conf` files.

Heh, I had `version = "3.8.3"` in at some point in [my bzlmod working branch](https://github.com/mbland/rules_scala/commits/bzlmod/), because Scalafmt failed without it. Then I took it out here because it didn't seem required. But I do like the explicitness of it.